polymesh-docs/primitives/multisig.mdx

```diff
@@ -1,110 +1,158 @@
+
+In order to administer the MultiSig, it is the Primary Key of the *Admin Identity* that must submit the relevant transactions.
+
+Assigning an *Admin Identity* can be done through a usual MultiSig proposal to `multisig::addAdmin` which is then approved in the usual way.
+
+![Proposing New Admin](../images/multisig/17-add_admin.png)
+
+![Approving New Admin](../images/multisig/18-approve_admin.png)
+
+The event confirming the new *Admin Identity* can then be observed.
+
+![Event For New Admin](../images/multisig/19-admin_event.png)
+
+Once the *Admin Identity* has been approved it can, for example, update the number of approvers needed from 2 to 1.
+
+![Updating Signers Required](../images/multisig/20-change_signers.png)
+
+![Updating Signers Required Event](../images/multisig/21-signers_event.png)
+
+## Removing Payer Identity
+
+On creation a MultiSig is assigned the identity of the calling key as its *Paying Identity*.
+
+It may be that you wish to remove this association, leaving the MultiSig to pay for its own transactions, from its POLYX balance (rather than relying on the primary key of the *Paying Identity* to pay for its associated transactions).
+
+A *Paying Identity* can be removed either by the MultiSig, or by the *Paying Identity* itself. In this example we do the former.
+
+![Remove Paying Identity](../images/multisig/22-remove_payer.png)
+
+![Remove Paying Identity Event](../images/multisig/23-remove_payer_event.png)
+
+## Changing Identity
+
+It may be that the MultiSig needs to be detached from its current identity, and attached to a new identity.
+
+In order to be detached from an identity, a MultiSig must have a POLYX balance of less than 1 POLYX. This is to avoid a large amount of POLYX being locked on a MultiSig which can no longer transact as a result of not being attached as a primary or secondary key to an identity.
+
+Once the balance has been decreased below 1 POLYX, the MultiSig can leave its current identity.
+
+![Leave Identity](../images/multisig/24-leave_identity.png)
+
+![Leave Identity Event](../images/multisig/25-leave_identity_event.png)
+
+In order to join a new identity, the new identity must issue a `JoinIdentity` authorisation to the MultiSig.
+
+![Join Identity](../images/multisig/26-join_identity.png)
+
+![Join Identity Event](../images/multisig/27-join_identity_event.png)
+
+The MultiSig can then accept this `JoinIdentity` authorisation. Note that when accepting a `JoinIdentity` authorisation, it is always the primary key of the identity that issued the `JoinIdentity` authorisation that pays for the transaction fee to accept the authorisation.
+
+To join a new identity, a MultiSig must use the dedicated extrinsic `multisig::approveJoinIdentity` - this ensures that payment is routed correctly (otherwise the transaction cannot complete as, in this instance, the MultiSig cannot pay for itself as it is not yet attached to an identity).
```

Comment:
I would recommend that users always try to do an atomic batch to leave & join a new DID.  The `approve_join_identity` should be for recovery of a MS, or when the new DID isn't currently setup/available.